packages/server-socket/src/command-processor.js

```diff
@@ -9,16 +9,27 @@ function createCommandProcessor({ Event, logger }) {
 function processCommand({ Event, logger }) {
   return sendEvent => command => {
     const helpers = { Event, logger, sendEvent, userId: command.userId };
+    const labels = setLabels(command);
     if (command.command === "ADD_TODO") {
-      return createEvent({ ...helpers, eventType: "ADDED_TODO", data: { ...command.data } });
+      return createEvent({ ...helpers, eventType: "ADDED_TODO", data: { ...command.data, labels } });
     } else if (command.command === "REMOVE_TODO") {
-      return createEvent({ ...helpers, eventType: "REMOVED_TODO", data: { ...command.data } });
+      return createEvent({ ...helpers, eventType: "REMOVED_TODO", data: { ...command.data, labels } });
     } else if (command.command === "CHANGE_TODO") {
-      return createEvent({ ...helpers, eventType: "CHANGED_TODO", data: { ...command.data } });
+      return createEvent({ ...helpers, eventType: "CHANGED_TODO", data: { ...command.data, labels } });
     }
   };
 }
 
+function setLabels(command) {
+  // We have to actively set labels to undefined if it isn't set due to
```

Comment:
It actually might make sense to have `labels: []` always instead of having to check whether `labels` is defined or not.